apps/hash-graph/Makefile.toml

```diff
@@ -5,39 +5,32 @@ description = "Installs the required yarn dependencies for generating the API cl
 extend = "yarn"
 cwd = "${REPO_ROOT}/apps/hash-graph/clients"
 
-[tasks.migrate-up]
-category = "Deploy"
-description = "Runs all migrations"
-extend = "docker"
-args = ["compose", "--env-file", "${REPO_ROOT}/apps/hash-graph/.env", "up", "migrate", "--no-recreate"]
-cwd = "${REPO_ROOT}/apps/hash-graph/deployment"
-
 [tasks.deployment-up]
 run_task = { name = ["docker-compose-deployment-up", "migrate-db"] }
 
 [tasks.docker-compose-deployment-up]
 private = false
 category = "Deploy"
 description = "Spins up the deployment environment"
-extend = "docker"
-args = ["compose", "--env-file", "../.env", "up", "--wait", "--build"]
-cwd = "${REPO_ROOT}/apps/hash-graph/deployment"
+extend = "yarn"
+args = ["external-services", "up", "postgres", "--wait"]
+cwd = "${REPO_ROOT}"
 
 [tasks.migrate-db]
 private = false
 category = "Deploy"
-extend = "run-task"
 description = "Run DB migrations using the Graph binary"
-cwd = "${REPO_ROOT}/apps/hash-graph"
-args = ["run", "--manifest-path", "./hash_graph/Cargo.toml", "--", "migrate"]
+extend = "yarn"
+args = ["external-services", "up", "graph-migrate", "--build"]
```

Comment:
We've gone from running the binary "natively" to running the migrations through `external-services` here, is that just for consistency and using the same setup throughout?

Reply:
I run the migrations natively in https://github.com/hashintel/hash/pull/1932